pom.xml

```diff
@@ -227,8 +227,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.1</version>
         <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
+          <source>1.8</source>
+          <target>1.8</target>
```

Comment:
I think the best bet right now is try to re-build ipfs jar with target 1.7 locally